Expensify/App
# Fix popovers after regression

cc @npsedhain @tgolen 

### Details
Fixing a regression caused by [this PR](https://github.com/Expensify/Expensify.cash/pull/2222)

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/2420

### Tests / QA Steps (Web/Desktop)
1. Hit the global create FAB, and verify that the `CreateMenu` appears in the correct position in the sidebar. It should fade in from the left and fade out to the left.
2. Hit the `+` icon in the `ReportActionCompose`. Verify that the `CreateMenu` appears in the correct position above the compose input. It should fade in from the bottom and fade out to the bottom.
3. Go to `Settings` -> `Profile` -> `Edit Photo`. Verify that the `CreateMenu` appears in the correct position above the "Edit Photo" button. It should fade in from the right and fade out to the right.
4. Right click on a message, and verify that the `ReportActionContextMenu` appears as it should in the location where the user clicks.
5. Open the emoji picker, make sure it looks normal (it shouldn't have any noticeable animation when it opens and closes).

### Tests / QA Steps (mWeb/iOS/Android)
1. Hit the following buttons, and verify that the `CreateMenu` appears as a bottom-docked modal:
    1. Global create FAB
    1. `+` in the `ReportActionCompose`
    1. `Settings` -> `Profile` -> `Edit Photo`
    1. Long-press a message.
    1. The emoji picker

### Tested On


### Screenshots

|_Menu_|Web|Mobile Web|Desktop|iOS|Android|
|---|---|---|---|---|---|
|Global Create|![image](https://user-images.githubusercontent.com/47436092/114932048-e9c7e880-9deb-11eb-8f81-45dfcc5be877.png)|![image](https://user-images.githubusercontent.com/47436092/114932625-90ac8480-9dec-11eb-9f0b-a313dab6b8ed.png)|![image](https://user-images.githubusercontent.com/47436092/114933113-2ea04f00-9ded-11eb-9e78-1ffcd51bc750.png)|![image](https://user-images.githubusercontent.com/47436092/114933567-a8d0d380-9ded-11eb-9a86-c940f114269d.png)|![image](https://user-images.githubusercontent.com/47436092/114934136-60fe7c00-9dee-11eb-94db-e05a841e89f1.png)|
|ReportActionCompose|![image](https://user-images.githubusercontent.com/47436092/114932207-15e36980-9dec-11eb-970d-cace8ca56c80.png)|![image](https://user-images.githubusercontent.com/47436092/114932675-9efaa080-9dec-11eb-8333-daf87ee6b87a.png)|![image](https://user-images.githubusercontent.com/47436092/114933154-395ae400-9ded-11eb-8eeb-9bd91af05310.png)|![image](https://user-images.githubusercontent.com/47436092/114933624-b6865900-9ded-11eb-8963-70c924c2e35f.png)|![image](https://user-images.githubusercontent.com/47436092/114934188-72e01f00-9dee-11eb-81ce-2bab34b40fac.png)|
|Edit Photo|![image](https://user-images.githubusercontent.com/47436092/114931957-ca30c000-9deb-11eb-8f17-77a1da099b2f.png)|![image](https://user-images.githubusercontent.com/47436092/114932941-fa2c9300-9dec-11eb-9aa9-25d63a395417.png)|![image](https://user-images.githubusercontent.com/47436092/114933191-44157900-9ded-11eb-8561-44f9cdac027c.png)|![image](https://user-images.githubusercontent.com/47436092/114933654-c1d98480-9ded-11eb-9a6e-2c0ad9f60784.png)|![image](https://user-images.githubusercontent.com/47436092/114934228-7f647780-9dee-11eb-9b60-94c2aa9eb910.png)|
|ReportActionContextMenu|![image](https://user-images.githubusercontent.com/47436092/114931992-d6b51880-9deb-11eb-95f9-bcaae1506e6b.png)|![image](https://user-images.githubusercontent.com/47436092/114932879-e2550f00-9dec-11eb-9926-7885003b2052.png)|![image](https://user-images.githubusercontent.com/47436092/114933245-4f68a480-9ded-11eb-9345-9869d1589190.png)|![image](https://user-images.githubusercontent.com/47436092/114933688-ce5ddd00-9ded-11eb-8841-d18b893b1d96.png)|![image](https://user-images.githubusercontent.com/47436092/114934299-960ace80-9dee-11eb-9f76-9e9f14530ffe.png)|
|Emoji Picker|![image](https://user-images.githubusercontent.com/47436092/114957473-f95c2700-9e15-11eb-97e1-558e29a0d1b8.png)|![image](https://user-images.githubusercontent.com/47436092/114957578-28729880-9e16-11eb-818b-67ea87e2139e.png)|![image](https://user-images.githubusercontent.com/47436092/114957708-74254200-9e16-11eb-9e8e-5329a476b2c2.png)|||
